.github/workflows/codedov.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Github CI
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches: "*"
```

Comment:
So this only works for develop branches, why not all of them?

Reply:
If you directly push to develop, yes.
But, we also let it run on all pull_requests.
Do you want to add more branches to the `push` section?

Reply:
No this is good!